frontend/src/lib/lemon-ui/LemonTable/LemonTable.tsx

```diff
@@ -52,8 +52,8 @@ export interface LemonTableProps<T extends Record<string, any>> {
     inset?: boolean
     /** An embedded table has no border around it and no background. This way it blends better into other components. */
     embedded?: boolean
-    /** Whether inner table borders should be shown. **/
-    borderedRows?: boolean
+    /** Whether a table background and inner borders should be shown. **/
```

Comment:
More clear to me if comments on boolean props describe what they do for the `true` case.
```suggestion
    /** Whether to hide the table background and inner borders. **/
```